### ISSUE TITLE
Feature extraction

### ISSUE THREAD

**Issue by Shaw0446** [increase24/awesome-sEMG-hand-gesture-recognition#1]
I have tried to extract the EMG characteristics of different channels, but how can they be normalized and reassembled into multidimensional inputs

**Comment by Zeng-Jia** [comment]
> I have tried to extract the EMG characteristics of different channels, but how can they be normalized and reassembled into multidimensional inputs

This repository is about deep learning approach for sEMG based gesture recognition, so feature extraction is not involved in this 
 repository. There is instruction to extract the features of sEMG in https://github.com/increase24/fatigue-sensitivity-USEMG.

I assume that you want to extract features from multiple channels as input to a machine learning classifier or neural network. In regard to normalization, in my opinion, uniform normalization across multiple channels for the same type of features is better. You can have a try. For more detailed configuration, maybe you can refer this paper <Surface-Electromyography-Based Gesture Recognition by Multi-View Deep Learning> (https://ieeexplore.ieee.org/abstract/document/8641445).

**Comment by Shaw0446** [comment]
> > I have tried to extract the EMG characteristics of different channels, but how can they be normalized and reassembled into multidimensional inputs
> 
> This repository is about deep learning approach for sEMG based gesture recognition, so feature extraction is not involved in this repository. There is instruction to extract the features of sEMG in https://github.com/increase24/fatigue-sensitivity-USEMG.
> 
> I assume that you want to extract features from multiple channels as input to a machine learning classifier or neural network. In regard to normalization, in my opinion, uniform normalization across multiple channels for the same type of features is better. You can have a try. For more detailed configuration, maybe you can refer this paper (https://ieeexplore.ieee.org/abstract/document/8641445).

Thanks for your reply, the idea for feature extraction came from this article, unfortunately there is still a lot of confusion in view construction.  It has always been difficult to achieve a high recognition rate in experiments without extracting features. I noticed that you use minmax to normalize all data in preprocessing. After data division, I normalize the training set first, and then share the most value.  Normalizing the test set, still did not meet expectations

**Comment by Zeng-Jia** [comment]
> 
I'm not the contributer of paper "Surface-Electromyography-Based Gesture Recognition by Multi-View Deep Learning" , so I can't answer you if the results do not meet your expectations. The author release their codes at
https://github.com/computer-animation-perception-group/sEMG-based-mvcnn, you may need to contact the authors about the details. I also have the plan to reproduce the MVCNN network, I'll get you known if I get progress.
